spec/services/page_builder_spec.rb

```diff
@@ -1,10 +1,8 @@
 require 'rails_helper'
 
 describe PageBuilder do
-
-  subject { PageBuilder.create_with_plugins(params) }
-
-  let(:params) {{ title: "Foo Bar", liquid_layout_id: template.id }}
+  let(:language) { create(:language) }
+  let(:params) {{ title: "Foo Bar", liquid_layout_id: template.id, language_id: language.id }}
```

Comment:
`params` here are only for creating the page, and are not part of the request body that's sent to the queue.